modules/apps/object/object-test/src/testFunctional/macros/ObjectNotifications.macro

```diff
@@ -180,6 +180,20 @@ definition {
 		MenuItem.clickNoError(menuItem = "Resend");
 	}
 
+	macro selectEntityAndCheckTerms {
+		Click(locator1 = "ObjectCustomViews#SELECT_AUTO_COMPLETE");
+
+		Click(
+			key_optionName = ${entityName},
+			locator1 = "ObjectAdmin#LAYOUT_ADD_FIELD_OPTION");
+
+		for (var entityTerms : list ${listOfTerms}) {
+			AssertElementPresent(
+				key_termName = ${entityTerms},
+				locator1 = "ObjectNotifications#ENTITY_TERMS");
+		}
```

Comment:
A couple of things:
- This is combining two things, a navigation and an assertion. Navigation macros should be their own separate entity so that they can be reused in other areas. For this use case specifically, I'm not sure if there's a lot of value in encapsulating this in a macro unless we have a lot of tests doing this.
- We shouldn't be including an assertion in a macro, that should be on the testcase level

Reply:
As I remember I create this macro to turn easy ensure that the criterias from [this](https://liferay.atlassian.net/browse/LPS-171625) story was been done. I have create it instead put directly on the testcase level to turn reusable in future tests and make easier for whoever was going to do it. So, should I:

1. Delete the macro and put the structure in the testcase level
2. Remove the assertion from the macro and put in testcase level
3. Leave as is
cc:@timpak

Reply:
If that macro is only being used in one test, #1. If the test is used in multiple tests, then #2.